src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,67 @@ redirects:
-
-  </LandingPageTile>
-
-  <LandingPageTile
-    title="Resolve an issue."
-    href="/docs/alerts/new-relic-alerts/reviewing-alert-incidents"
-    icon="fe-tool"
-  >
-     [Acknowledge an incident](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/acknowledge-alert-incidents) after it occurs, or [close a violation manually](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/close-violations-manually) as needed.
-  </LandingPageTile>
-</LandingPageTileGrid>
+
+In the New Relic UI, Alerts & AI is divided into several sections, each unlocking different features. Check out the section explanations below to learn how these features can meet your needs.
+
+## **Analyze** your issues, incidents, and anomalies [#analyze]
+
+Use the analyze features to scan for critical problems affecting your systems, via alerts violations, issues, incidents, and anomalies.
+
+* **[Overview](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page)**: Find your most recent open, closed, and muted violations.
+* **Issues and activity**: Review your most recent [issues](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#issues), [incidents](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#incidents), and [anomalies](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-ai-overview-page/#anomalies).
+
+## **Detect** problems in your system [#detect]
+
+Use the detect features to define the conditions in your system you want to be notified about, as well as to configure how anomalies are detected.
+
+* **[Alert conditions (policies)](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/)**: Create new policies. Also, lists the policies where your conditions are created and organized. 
+* **[Anomaly detection](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/)**: Manage your anomaly detection configurations. An anomaly is a behavioral trend that doesn't match the historical data for your system. You can add or edit the configurations here.
+
+## **Correlate** the data coming into your system [#correlate]
+
+Use the correlate features to define your data sources and to review and configure the logic that groups your incidents.
+
+* **[Sources](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources)**: Manage the data input sources you've chosen to analyze and be notified about. You can add new sources or configure existing ones.
+* **[Decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/)** Review your correlated incidents. You can also review, edit, and add decisions.
```

Comment:
@paperclypse, looks like you need a colon after Decisions. Also, since decisions is so unintuitive as a label, is there some blurb you could add to help users understand what these are ? I see that decisions are correlated incidents, but is there a way to make this more explicit?